opre_ops/ops_site/admin.py

```diff
@@ -50,7 +50,7 @@ class CANAmount(admin.ModelAdmin):
                     "team_leader", "can_division", "notes")
 
     def can_display_name(self, obj):
-        return obj.can.number + " (" + obj.can.nickname + ")" + " - " + obj.fiscal_year
+        return obj.can.number + " (" + obj.can.nickname + ")" + " - " + str(obj.fiscal_year)
```

Comment:
[thought][non-blocking] There's something in Python 3 that called [f-string](https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/).  It makes the string looks cleaner.

Reply:
Fwiw I like this non-blocking suggestion, f-strings make interpolated strings like this much easier to read.